### DISCUSSION TITLE
Define custom error view

### DISCUSSION THREAD

**Discussion by Ghoelian** [spatie/laravel-ignition#27] [Q&A] [answered]
Is there any way to define a custom error view? I couldn't find anything related to this in the docs.

We have CSP headers with unsafe-inline disabled, so the Ignition error page shows up blank, since it seems all of the info and styling is applied through inline style and script blocks.

**Comment by freekmurze** [comment, marked as answer]
There's no support for custom views. We'd accept a PR that would solve the CSP problem in Ignition.
